README.md

```diff
@@ -1,11 +1,11 @@
 # Bitmovin Player UI [![npm version](https://badge.fury.io/js/bitmovin-player-ui.svg)](https://badge.fury.io/js/bitmovin-player-ui) [![Build Status](https://travis-ci.org/bitmovin/bitmovin-player-ui.svg?branch=master)](https://travis-ci.org/bitmovin/bitmovin-player-ui)
 The Bitmovin Adaptive Streaming Player UI
 
-Read more about the usage, as well as other important information on Bitmovin's Adaptive Streaming Player itself at https://bitmovin.com/ and https://bitmovin.com/player-documentation/.
+Read more about the usage,  along with other important information about the Bitmovin's Adaptive Streaming Player at https://bitmovin.com/ and https://bitmovin.com/player-documentation/.
```

Comment:
I think the `the` shouldn't be there (or the `'s` in `Bitmovin's`):
```suggestion
Read more about the usage, along with other important information about Bitmovin's Adaptive Streaming Player at https://bitmovin.com/ and https://bitmovin.com/player-documentation/.
```

Reply:
- `Bitmovin's Adaptive Streaming Player` should actually now be simply called `Bitmovin Player`.
- The up-to-date URL is https://bitmovin.com/docs/player

```suggestion
Read more about the usage, along with other important information about the Bitmovin Player at https://bitmovin.com/ and https://bitmovin.com/docs/player.
```